src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Migrations/28.diff.sql

```diff
@@ -429,6 +431,8 @@ BEGIN
     )
     WITH (DATA_COMPRESSION = PAGE, ONLINE=ON)
```

Comment:
We should be able to accomplish the same thing with the hint `DROP_EXISTING=ON`

Reply:
I think it won't work if the index is not created yet

Reply:
Actually, it's probably not going to work the way I thought it would. Supposedly it doesn't work on an initial creation.

Reply:
Yes